lib/pack_rb/sub_commands.rb

```diff
@@ -10,16 +10,63 @@ class SubCommands
     include PackRb::SubCommands::Inspect
     include PackRb::SubCommands::Validate
 
+    # Execute a Packer command via {#run_cmd_stream_output}
+    #
+    # @param opts [Hash] options passed to the Packer class
+    #
+    # @return [Array] - stdout [String], stderr [String], exit code [Fixnum]
     def execute(opts)
       cmd = opts[:cmd]
       tpl = opts[:tpl]
 
-      out, err, status = Open3.capture3("#{cmd} -", stdin_data: tpl)
+      out, err, status = run_cmd_stream_output("#{cmd} -", tpl)
```

Comment:
This seems redundant. I'd go with the Ruby convention that RuboCop enforces on this one.